targettests/execution/bug_qubit.cpp

```diff
@@ -9,6 +9,7 @@
 // This code is from Issue 251.
 
 // clang-format off
+// RUN: nvq++ %cpp_std --target anyon                    --emulate %s -o %t && %t | FileCheck %s
```

Comment:
@bmhowe23 This file is symbolically linked in the specific target directories (and this PR is following that). But the RUN lines just test all manner of targets in the file and not just ionq for the ionq backend, for example. Is that what we want?

Reply:
To a first order, yes, I think that is what we want. Adding the `RUN` entries w/ `--emulate` to these files ensure that they run locally and produce the desired results with the specific target. Symbolically linking them into the specific target directories makes them run withOUT `--emulate`, so they actually submit to the true target. Those are two different tests, and I think they are both valid.

However, if some of the local tests are taking too long, we could consider making the local tests more limited on a case by case basis.

Reply:
this comment seems resolved?